bigtop-packages/src/charm/spark/layer-spark/reactive/spark.py

```diff
@@ -38,13 +38,15 @@ def report_status():
                            'yarn execution mode not available')
         return
 
-    if mode == 'standalone' and is_state('leadership.is_leader'):
+    if mode == 'standalone' and is_state('zookeeper.ready'):
+        mode = mode + " - HA"
+    elif mode == 'standalone' and is_state('leadership.is_leader'):
```

Comment:
Is there ever a case where zookeeper.ready is set and leadership.is_leader is also set? If so, is it correct to ignore the leadership.is_leader state, as this code currently does?